homeassistant/components/rainmachine/switch.py

```diff
@@ -197,9 +197,9 @@ async def async_setup_entry(
         ),
         ("stop_program", {}, "async_stop_program"),
         ("stop_zone", {}, "async_stop_zone"),
```

Comment:
```suggestion
        ("stop_program", None, "async_stop_program"),
        ("stop_zone", None, "async_stop_zone"),
```
Should these be updated as well?

Reply:
Yes, absolutely, thanks @cdce8p 👍